pkgs/applications/audio/nuclear/default.nix

```diff
@@ -1,33 +1,35 @@
-{
-  appimageTools,
-  lib,
-  fetchurl,
-}: let
+{ appimageTools
+, lib
+, fetchurl
+,
```

Comment:
```suggestion
```

Reply:
done automatically by nixpkgs-fmt as far as I'm aware

Reply:
Nixpkgs-fmt just moves commas to start lines. There was a trailing comma in original inputs that should be removed.